inst/rmarkdown/resources/html/print_button.html

```diff
@@ -0,0 +1,7 @@
+<button id="print" style="left:36%;position:relative;background-color: #92bcbc94;">
+  <i class="glyphicon glyphicon-print" type="submit" onclick="window.PagedPolyfill.preview().then(() => {document.body.style.cssText='min-width: unset !important'; window.print();})" style= "padding-left: 5px; color: #cf581b;">
```

Comment:
On ne peut pas laisser ça comme ça, c'est trop illisible. Tu peux accepter la modification ci-dessous ainsi que celle-ci : 

Il faut aussi déplacer le style inline dans une feuille de style.

```suggestion
  <i class="glyphicon glyphicon-print" type="submit" onclick="printWithPagedJS()" style= "padding-left: 5px; color: #cf581b;">
```